scripts/build/termux_step_get_dependencies.sh

```diff
@@ -133,11 +148,12 @@ termux_run_build-package() {
 		fi
 	fi
 	TERMUX_BUILD_IGNORE_LOCK=true ./build-package.sh \
- 		$(test "${TERMUX_INSTALL_DEPS}" = "true" && echo "-I" || echo "-s") \
- 		$({ test "${TERMUX_FORCE_BUILD}" = "true" && test "${TERMUX_FORCE_BUILD_DEPENDENCIES}" = "true"; } && echo "-F") \
- 		$(test "${TERMUX_PKGS__BUILD__RM_ALL_PKG_BUILD_DEPENDENT_DIRS}" = "true" && echo "-r") \
-   		$(test "${TERMUX_WITHOUT_DEPVERSION_BINDING}" = "true" && echo "-w") \
-     		--format $TERMUX_PACKAGE_FORMAT --library $set_library "${PKG_DIR}"
+		$(test "${TERMUX_INSTALL_DEPS}" = "true" && echo "-I" || echo "-s") \
+		$({ test "${TERMUX_FORCE_BUILD}" = "true" && test "${TERMUX_FORCE_BUILD_DEPENDENCIES}" = "true"; } && echo "-F") \
```

Comment:
No need for `{` and `}` here, in the case if any of tests fail `echo` will not be called.

Reply:
Only indent was changed here, the `{}` wrapping was done in https://github.com/termux/termux-packages/commit/9159eab18ae69a102d56ca3b24cec779f1e6f20c and it makes it clear under what conditions should `-F` be passed, related to https://www.shellcheck.net/wiki/SC2015